core/src/main/java/com/graphhopper/search/StringIndex.java

```diff
@@ -1,337 +0,0 @@
-/*
- *  Licensed to GraphHopper GmbH under one or more contributor
- *  license agreements. See the NOTICE file distributed with this work for
- *  additional information regarding copyright ownership.
- *
- *  GraphHopper GmbH licenses this file to you under the Apache License,
- *  Version 2.0 (the "License"); you may not use this file except in
- *  compliance with the License. You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
-package com.graphhopper.search;
-
-import com.graphhopper.storage.DataAccess;
-import com.graphhopper.storage.Directory;
-import com.graphhopper.util.BitUtil;
-import com.graphhopper.util.Constants;
-import com.graphhopper.util.GHUtility;
-import com.graphhopper.util.Helper;
-
-import java.util.*;
-
-/**
- * @author Peter Karich
- */
-public class StringIndex {
-    private static final long EMPTY_POINTER = 0, START_POINTER = 1;
```

Comment:
Unfortunately git no longer recognizes StringIndex.java and EdgeKVStorage.java to be the same file, even though they are still quite similar, and the same for the corresponding tests. It might be worth to first do (only) the renaming in master and then merging this PR. This way there will be a commit that still allows seeing the actual changes.